lib/linter-eslint.coffee

```diff
@@ -78,6 +87,9 @@ module.exports =
             console.warn '[Linter-ESLint] ESlint rules directory does not exist in your fs'
             console.warn error.message
 
+        # Add file extensions option
+        options.extnensions = atom.config.get('linter-eslint.fileExtensions').split(',')
```

Comment:
Is this even necessary? We are running executeOnText() instead of running on a file.